ansible_roles/wordpress/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: enable epel
```

Comment:
i would recomend to split this into a separate repos' role, it's easyer to maintain stuff in the future. Or you will endup enabling/disabling repos multiple times from multiple roles. this is just a recomandation feel free to ignore it.